File: URI/Java/TempoDeJogo.java
import java.util.Scanner;
import java.lang.Math;

public class TempoDeJogo {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int ini, fim, dur;
        ini = input.nextInt();
        fim = input.nextInt();
        dur = fim - ini;
        if (dur == 0) {
            dur = 24;
        } else if (dur < 0) {
            dur += 24;
        }
        System.out.println("O JOGO DUROU "+dur+" HORA(S)");
        input.close();
    }
}

File: URI/Python/TempoDeJogo.py
inf = [int(x) for x in input().split()]
ini = inf[0]
fim = inf[1]
dur = fim -ini

if dur == 0:
    dur = 24
elif dur < 0:
    dur+=24
print("O JOGO DUROU {} HORA(S)".format(dur))